integ/components/deadline/deadline_05_secretsManagement/test/deadline_05_secretsManagement.test.ts

```diff
@@ -3,12 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { BlockList } from 'net';
 import { CloudFormation } from '@aws-sdk/client-cloudformation';
 import { SecretsManager } from '@aws-sdk/client-secrets-manager';
-import {
-  CidrBlock,
-  NetworkUtils,
-} from 'aws-cdk-lib/aws-ec2/lib/network-util';
+// import {
```

Comment:
I guess we can remove these instead of comment-out?

Reply:
Ahhhh, thanks for catching that!